clients/pkg/logentry/stages/pipeline.go

```diff
@@ -76,10 +93,20 @@ func (p *Pipeline) Run(in chan Entry) chan Entry {
 		}
 		return e
 	})
+
 	// chain all stages together.
 	for _, m := range p.stages {
 		in = m.Run(in)
 	}
+
+	if InspectWithId {
+		// add a last stage to remove the inspect id
+		/*in = RunWith(in, func (e Entry) Entry {
+			delete(e.Extracted, "InspectId")
+			return e
+		})*/
```

Comment:
What do you want to do with this?

Reply:
oh, that's a left over, I first thought that I have to remove the entry from Extracted, so that I don't polute the labels, but afterwards came to the conclusion I need the id also in the logline

=> will be removed